lldb/source/Plugins/ExpressionParser/Swift/SwiftUserExpression.cpp

```diff
@@ -693,7 +693,7 @@ bool SwiftUserExpression::Parse(DiagnosticManager &diagnostic_manager,
     return error("could not create a Swift AST context");
 
   if (m_swift_ast_ctx->HasFatalErrors()) {
-    m_swift_ast_ctx->PrintDiagnostics(diagnostic_manager);
+    m_swift_ast_ctx->PrintDiagnostics(diagnostic_manager, false);
```

Comment:
What's the meaning of the second argument? It would be helpful to add an inline comment `/*ArgName=*/`